package.json

```diff
@@ -423,7 +430,8 @@
               "name": "Untitled",
               "mainClass": "???",
               "args": [],
-              "jvmOptions": []
+              "jvmOptions": [],
+              "env": []
```

Comment:
Same as above, shouldn't this be `{}` now?

Reply:
And fixed!